git_machete/generated_docs.py

```diff
@@ -7,12 +7,12 @@
 long_docs: Dict[str, str] = {
```

Comment:
One more thing... for consistency with `handmade_short_docs.py`, maybe rename the file to `generated_long_docs.py`